package.json

```diff
@@ -84,7 +86,7 @@
     "test:acceptance": "cd tests/acceptance; webpack --bail; NODE_ENV=production webpack --bail",
     "release": "semantic-release pre && npm publish && semantic-release post",
     "commitmsg": "validate-commit-msg",
-    "precommit": "npm run lint && npm run flow && npm run test && npm run test:acceptance"
```

Comment:
* please bring back `build` folder. The `npm run build` command should still generate a `build` folder, that's the production build.
* please rename `builds` to `build` like all other repositories we have